main.js

```diff
@@ -1,4 +1,4 @@
-//--------------------/array variables/----------------------//
+//--------------------/Data Model/----------------------//
```

Comment:
I would consider moving all of this data into a data.js file just to clean up your main.js

Reply:
Yes, great idea!